api/src/features/metrics/metrics_feature.py

```diff
@@ -0,0 +1,20 @@
+from fastapi import APIRouter
+from fastapi.responses import PlainTextResponse
+
+from config import config
+
+router = APIRouter(tags=["metrics"])
+
+
+class Metrics:
+    MULTIFLASH_CALCULATION_COUNT: int = 0
+    FETCH_COMPONENTS_COUNT: int = 0
+
+
+metrics = Metrics()
+
+
+@router.get("/metrics", response_class=PlainTextResponse, summary="Collect application metrics", response_model=str)
+async def get() -> str:
+    return f"""mercury_calculations_count {{env={config.ENVIRONMENT}}} {metrics.MULTIFLASH_CALCULATION_COUNT}\n
```

Comment:
Why triple quotes?

Reply:
For easier reading and writing of multi line strings.
Would be uglier to use many brackets and +'es IMO